nb-extension/js/main.js

```diff
@@ -20,11 +20,25 @@ define([
         window.console[method] = window.console[method] || window.console.log;
     });
     
-    init({
-        namespace: Jupyter,
-        events: events,
-        WidgetManager: widgetManager.WidgetManager,
-        WidgetModel: ipywidget.WidgetModel
+    // setup package name; required for later use by jupyter-js-widgets
+    requirejs({
```

Comment:
This was one of the parts I was most apprehensive about, having an extension muck around with the require.js configuration of the notebook. As I mentioned in the other PR, don't worry about this. I have dashboard server working with your changes, so I think we should stick with that.